lib/sablon/parser/mail_merge.rb

```diff
@@ -111,13 +111,16 @@ def parse_fields(xml)
       end
 
       private
+
       def build_complex_field(node)
         possible_field_node = node.parent
         field_nodes = [possible_field_node]
         while possible_field_node && possible_field_node.search(".//w:fldChar[@w:fldCharType='end']").empty?
           possible_field_node = possible_field_node.next_element
           field_nodes << possible_field_node
         end
+        # skip instantiation if no end tag
+        return if field_nodes.last.nil?
```

Comment:
small nitpick but instead of an early return can we formulate it like this:

```ruby
# skip instantiation if no matching end tag was found
if field_nodes.last
  ComplexField.new(field_nodes)
end
```